proof/refine/ARM_HYP/Interrupt_R.thy

```diff
@@ -787,6 +787,11 @@ lemma virqSetEOIIRQEN_eq[simp]:
   unfolding virqSetEOIIRQEN_def ARM_HYP_A.virqSetEOIIRQEN_def
   by (rule ext) auto
 
+(* FIXME arch-split: RAF MOVE *)
+lemma not_pred_tcb':
+  "(\<not>pred_tcb_at' proj P t s) = (\<not>tcb_at' t s \<or> pred_tcb_at' proj (\<lambda>a. \<not>P a) t s)"
+  by (auto simp: pred_tcb_at'_def obj_at'_def)
```

Comment:
That one is probably worth moving before we merge.

Reply:
Will do.